test/MStrategy.test.ts

```diff
@@ -1455,6 +1463,533 @@ contract<MStrategy, DeployOptions, CustomContext>("MStrategy", function () {
+                    );
+
+                    const address =
+                        await mockMStrategy.callStatic.createStrategy(
+                            this.params.tokens,
+                            this.params.erc20Vault,
+                            this.params.moneyVault,
+                            this.params.fee,
+                            this.params.admin
+                        );
+                    await mockMStrategy.createStrategy(
+                        this.params.tokens,
+                        this.params.erc20Vault,
+                        this.params.moneyVault,
+                        this.params.fee,
+                        this.params.admin
+                    );
+                    let subject = await ethers.getContractAt(
+                        "MockMStrategy",
+                        address
+                    );
+
+                    let erc20VaultNft = await this.vaultRegistry.nftForVault(
+                        this.params.erc20Vault
+                    );
+                    let moneyVaultNft = await this.vaultRegistry.nftForVault(
+                        this.params.moneyVault
+                    );
+
+                    await this.vaultRegistry
+                        .connect(this.admin)
+                        .adminApprove(this.params.erc20Vault, moneyVaultNft);
+                    await this.vaultRegistry
+                        .connect(this.admin)
+                        .adminApprove(subject.address, moneyVaultNft);
+
+                    await this.vaultRegistry
+                        .connect(this.admin)
+                        .adminApprove(this.params.moneyVault, erc20VaultNft);
+                    await this.vaultRegistry
+                        .connect(this.admin)
+                        .adminApprove(subject.address, erc20VaultNft);
+
+                    const oracleParams: OracleParamsStruct = {
+                        oracleObservationDelta: 101,
+                        maxTickDeviation: 50,
+                        maxSlippageD: Math.round(0.1 * 10 ** 9),
+                    };
+                    const ratioParams: RatioParamsStruct = {
+                        tickMin: 198240 - 5000,
+                        tickMax: 198240 + 5000,
+                        erc20MoneyRatioD: Math.round(0.1 * 10 ** 9),
+                        minErc20MoneyRatioDeviationD: Math.round(
+                            0.01 * 10 ** 9
+                        ),
+                        minTickRebalanceThreshold: 180,
+                        tickNeighborhood: 60,
+                        tickIncrease: 180,
+                    };
+
+                    await subject
+                        .connect(this.mStrategyAdmin)
+                        .setRatioParams(ratioParams);
+                    await subject
+                        .connect(this.mStrategyAdmin)
+                        .setOracleParams(oracleParams);
+
+                    await subject
```

Comment:
Need checks for two event called [here](https://github.com/mellow-finance/mellow-vaults/blob/main/contracts/strategies/MStrategy.sol#L293) and [here](https://github.com/mellow-finance/mellow-vaults/blob/main/contracts/strategies/MStrategy.sol#L297) (Event is called on the line https://github.com/mellow-finance/mellow-vaults/blob/main/contracts/vaults/IntegrationVault.sol#L117)